pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -0,0 +1,128 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	v1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	ScrapeConfigsKind   = "ScrapeConfig"
+	ScrapeConfigName    = "scrapeconfigs"
+	ScrapeConfigKindKey = "scrapeconfig"
+)
+
+// Target represents a target for Prometheus to scrape
+// +kubebuilder:validation:MinLength:=1
+// +kubebuilder:validation:Pattern:="[^/]+"
+type Target string
+
+// +genclient
+// +k8s:openapi-gen=true
+// +kubebuilder:resource:categories="prometheus-operator",shortName="scfg"
+// +kubebuilder:storageversion
+
+// ScrapeConfig defines a namespaced Prometheus scrape_config to be aggregated across
+// multiple namespaces into the Prometheus configuration.
+type ScrapeConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ScrapeConfigSpec `json:"spec"`
+}
+
+// DeepCopyObject implements the runtime.Object interface.
+func (l *ScrapeConfig) DeepCopyObject() runtime.Object {
+	return l.DeepCopy()
+}
+
+// ScrapeConfigList is a list of ScrapeConfigs.
+// +k8s:openapi-gen=true
+type ScrapeConfigList struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard list metadata
+	// More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata
+	metav1.ListMeta `json:"metadata,omitempty"`
+	// List of ScrapeConfigs
+	Items []*ScrapeConfig `json:"items"`
+}
+
+// DeepCopyObject implements the runtime.Object interface.
+func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
+	return l.DeepCopy()
+}
+
+// ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
+// +k8s:openapi-gen=true
+type ScrapeConfigSpec struct {
+	// StaticConfigs list of labeled statically configured targets for this job.
+	StaticConfigs []StaticConfig `json:"staticConfigs,omitempty"`
+	// FileSDConfigs list of file service discovery configurations.
+	FileSDConfigs []FileSDConfig `json:"fileSDConfigs,omitempty"`
+	// HTTPSDConfigs list of HTTP service discovery configurations.
+	HTTPSDConfigs []HTTPSDConfig `json:"httpSDConfigs,omitempty"`
+	// RelabelConfigs to apply to samples before scraping.
+	// Prometheus Operator automatically adds relabelings for a few standard Kubernetes fields.
+	// The original scrape job's name is available via the `__tmp_prometheus_job_name` label.
+	// More info: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
+	RelabelConfigs []*v1.RelabelConfig `json:"relabelings,omitempty"`
```

Comment:
```suggestion
	// More info: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
	// +optional
	RelabelConfigs []*v1.RelabelConfig `json:"relabelings,omitempty"`
```

Reply:
Thanks! Fixed in 576480855c09a6423659447eb6b9da15261a753a